maritalk/resources/local.py

```diff
@@ -0,0 +1,157 @@
+import os
+import time
+import shutil
+import atexit
+import subprocess
+from pathlib import Path
+from typing import List, Dict
+import requests
+from requests.exceptions import ConnectionError
+
+
+class MariTalkLocal:
+    def __init__(self, host: str = "localhost", port: int = 8080):
+        self.api_url = f"http://{host}:{port}"
+        self.process = None
+
+    def start_server(
+        self,
+        license: str,
+        port: int = 8080,
+        bin_path: str = f"{Path.home()}/bin/maritalk",
+    ):
+        if not os.path.exists(bin_path):
+            os.makedirs(os.path.dirname(bin_path), exist_ok=True)
+            print("Downloading `maritalk`...")
```

Comment:
```suggestion
            print("Downloading maritalk...")
```
Is there a need for this uncommon quote style, i.e., `?